usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -186,18 +189,19 @@ if automatic_recovery ; then
     done 2>&1
 fi
 
-# Launch rear recover automatically in unattended mode
-# i.e. with automated reboot after successful 'rear recover':
+# In unattended mode launch "rear --non-interactive recover" automatically 
+# i.e. with automated "reboot" (reboot_command) after successful recovery:
 if unattended_recovery ; then
+    timeout="${USER_INPUT_INTERRUPT_TIMEOUT:-30}"
```

Comment:
again, making `default.conf` obligatory would make this simpler

Reply:
yes